_controllers/controllers.js

```diff
@@ -50,4 +51,27 @@ exports.getReviewCommentsByID = (req,res,next) => {
         .catch((err)=>{
             next(err);
         })
+}
+
+exports.updateReviewVotes = (req,res,next) => {
+    const { review_id } = req.params;
+    const { inc_votes } = req.body;
+
+    if (!inc_votes) {
+        next({status: 400, msg: "Invalid Input: missing inc_votes"})
+    }
+
+    const review_check_promise = checkReviewExists(review_id)
+    const update_votes_promise = insertNewVotes(review_id, inc_votes)
+
+    Promise.all([update_votes_promise, review_check_promise])
```

Comment:
Love that youre using a Promise.all to do these multiple queries but we actually dont need to! In your `insertNewVotes` function you use `RETURNING *` and you can check how many rows were updated from this. If it was 0, then the review does not exist and we can reject the promise there instead, saving us a query